powerspikegg/computation_models/match/BUILD

```diff
@@ -13,6 +13,7 @@ py_binary(
     srcs = ["train_graph.py"],
     deps = [
         ":train",
+        "//powerspikegg/computation_models/fetcher",
```

Comment:
Let's actually... just move fetcher here. This is something only used by this part of the program, so it makes more sense to group it.